openreferee_server/operations.py

```diff
@@ -136,3 +137,29 @@ def process_pdf(file, session, upload_endpoint):
                 files={"file": (file["filename"], buf, file["content_type"])},
             )
             return r.json()
+
+
+def process_accepted_revision(event, revision):
+    publish = False
+    session = setup_requests_session(event.token)
+    available_tags = get_event_tags(session, event)
+    text = "This revision has been accepted but not published yet."
+    if revision["comment"] == "publish":
+        text = "This revision has been accepted for publishing."
+        publish = True
+    session.post(
+        revision["external_create_comment_url"], json={"text": text, "internal": True},
+    )
+    return dict(
+        publish=publish, tags=[available_tags["QA_APPROVED"]["id"]] if publish else []
+    )
+
+
+def process_revision(event, revision, action):
+    session = setup_requests_session(event.token)
+    available_tags = get_event_tags(session, event)
+    # TODO: return an internal comment via `comments=...`
+    # this should probably be the same structure we use when POSTing a comment,
+    # not just a string. e.g. this in this particular example:
+    # {"text": f"This revision has been reviewed ({action})", "internal": True}
```

Comment:
just leaving a comment to highlight this ;) also mentioned in the PR comments on the indico side